portability-core/pom.xml

```diff
@@ -278,6 +278,19 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!--Provides compile time bindings for slf4j -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.8.0-beta0</version>
```

Comment:
FWIW, Google has version 1.7.14 imported (for both libraries)

Reply:
Is there any problem with just leaving them on latest here?

Reply:
As per off line discussion, updated to 1.7.14